src/typography/index.js

```diff
@@ -240,10 +252,29 @@ export const Paragraph4 = React.forwardRef<BlockPropsT, HTMLElement>(
     />
   ),
 );
-Paragraph4.displayName = 'Paragraph4';
+ParagraphXSmall.displayName = 'ParagraphXSmall';
 
 // Aliases for backwards compatability
-export const Display = Display1;
+export const Display = DisplayLarge;
```

Comment:
can you add a comment like `TODO(v10): remove`, so we won't miss it when releasing the next major?

Reply:
Got it, thx!